### ISSUE TITLE
add  OCR for manhwa and manhua  and webtoon  [enhancements]

### ISSUE THREAD

**Issue by wolfkingal2000** [blueaxis/Poricom#70]
Add Chinese and Korea and english to OCR model thanks

**Comment by blueaxis** [comment]
EN, KR and CN is supported using Tesseract models. Or do you mean adding support using MangaOCR model? If so, that is not possible since you need to train another model for that. See [this issue](https://github.com/kha-white/manga-ocr/issues/24) for more info.

**Comment by wolfkingal2000** [comment]
> EN, KR and CN is supported using Tesseract models. Or do you mean adding support using MangaOCR model? If so, that is not possible since you need to train another model for that. See [this issue](https://github.com/kha-white/manga-ocr/issues/24) for more info.

Yes add  model like Manga OCR model  in JP real good job and about Tesseract   I don't know  how installed I try to pip but don't work if you can to add automate to software can be real good I mean download and installed in software

**Comment by blueaxis** [comment]
Have you read the issue I've linked above? As mentioned, it is not possible to add another language in MangaOCR, you need another model for that. This app (Poricom) is only a GUI wrapper and I didn't train the models used for text detection or translation.

As for Tesseract, what OS are you using? If you are on Windows you don't have to install anything, just download the zip file under releases. If you're on another OS or if you want to set it up locally, follow the instructions in the [README](https://github.com/blueaxis/Poricom#development-setup).
